install.sh

```diff
@@ -1,17 +1,25 @@
 #!/bin/sh
 set -e
 if [ "$1" != "" ]; then
-    mkdir -p $1/tools/agbcc
-    mkdir -p $1/tools/agbcc/bin
-    mkdir -p $1/tools/agbcc/include
-    mkdir -p $1/tools/agbcc/lib
-    cp agbcc $1/tools/agbcc/bin/
-    cp old_agbcc $1/tools/agbcc/bin/
-    cp agbcc_arm $1/tools/agbcc/bin/
-    cp -R libc/include $1/tools/agbcc/ #drop include, because we don't want include/include
-    cp ginclude/* $1/tools/agbcc/include/
-    cp libgcc.a $1/tools/agbcc/lib/
-    cp libc.a $1/tools/agbcc/lib/
+	if [ -d "$1" ]; then
+		mkdir -p $1/tools/agbcc
+		mkdir -p $1/tools/agbcc/bin
+		mkdir -p $1/tools/agbcc/include
+		mkdir -p $1/tools/agbcc/lib
+		cp agbcc $1/tools/agbcc/bin/
+		cp old_agbcc $1/tools/agbcc/bin/
+		cp agbcc_arm $1/tools/agbcc/bin/
+		cp -R libc/include $1/tools/agbcc/ #drop include, because we don't want include/include
+		cp ginclude/* $1/tools/agbcc/include/
+		cp libgcc.a $1/tools/agbcc/lib/
+		cp libc.a $1/tools/agbcc/lib/
+	else
+		if [ -d "../$1" ]; then
+			echo "Target directory does not exist. Did you mean to do \"./install.sh ../$1\"?"
+		else
+			echo "Target directory does not exist. If you aren't familiar with relative paths, make sure that agbcc and $1 are in the same directory, and run \"./install.sh ../$1\" again."
```

Comment:
`../` in `and run \"./install.sh ../$1\" again.` is redundant, e.g. `./install.sh ../badpath` would give the following message:
```
Target directory does not exist. If you aren't familiar with relative paths, make sure that agbcc and ../badpath are in the same directory, and run "./install.sh ../../badpath" again.
```

Reply:
oh right doy

Reply:
Fixed, please see the new script.